package.json

```diff
@@ -29,53 +26,38 @@
     "pug": "^2.0.4",
     "socket.io": "^2.3.0",
     "superagent": "^3.8.3",
-    "tinycolor2": "^1.4.1"
+    "tinycolor2": "^1.4.1",
+    "node-sass": "^4.12.0"
```

Comment:
I moved node-sass to deps on purpose so that we don't need extra stuff on Heroku.